packages/zoe/src/contractSupport/zoeHelpers.js

```diff
@@ -174,19 +175,23 @@ export const makeZoeHelpers = zcf => {
      * @property {TODO} [exit]
      *
      */
-    inviteAnOffer: (options = {}) => {
-      const {
-        offerHook = () => {},
-        customProperties = undefined,
-        expected = undefined,
-      } = options;
+    inviteAnOffer: ({
+      offerHook = () => {},
+      inviteDesc,
+      customProperties = undefined,
+      expected = undefined,
```

Comment:
I think `expected` should be required, since there is no reason to use this helper if `expected` is undefined.